### PR TITLE
NPM Workflows

### PR DESCRIPTION
**Pull request by WebRuin**
## Discord username (optional) please include so we can attribute you with our Contributor role (like so elvis#4747)
Tithos

## Description of changes (updated or new workflows)
I added a few NPM workflows for managing node_modules folders.

### CODE REVIEW

File: specs/npm/find_and_delete_node_modules_in_a_folder.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Find and delete node_modules in a folder
+command: find . -name 'node_modules' -type d -prune -print -exec rm -rf '{}' \;
+tags:
+  - npm
+description: Finds and deletes all node_modules in a folder
+arguments: []
+source_url: "https://stackoverflow.com/questions/22675725/find-unused-npm-packages-in-package-json"
```
Comment (alokedesai) [inline specs/npm/find_and_delete_node_modules_in_a_folder.yaml:8]: Is this source url correct for these workflows? I don't see the corresponding commands mentioned within the stackoverflow post

**Review by alokedesai — COMMENTED** [review]
Thanks for contributing! Just a few small comments to clarify my understanding!

File: specs/npm/look_for_all_directories_with_node_modules_and_show_them.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Finds and all node_modules in a folder
```
[the remark this answers is not in the document]
Comment (WebRuin) [reply specs/npm/look_for_all_directories_with_node_modules_and_show_them.yaml]: fixed

**Review by alokedesai — COMMENTED** [review]
One more comment! Besides that, it should be good to go :)

File: specs/ssh/copy_ssh_key_for_github.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: Copy SSH key for GitHub
+command: "pbcopy < ~/.ssh/id_ed25519.pub"
+tags:
+  - npm
```
[the remark this answers is not in the document]
Comment (WebRuin) [reply specs/ssh/copy_ssh_key_for_github.yaml:5]: sure

File: specs/ssh/generate_new_ssh_key_for_github.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+name: Generate new SSH key for GitHub
+command: "ssh-keygen -t ed25519 -C \"{{email}}\""
+tags:
+  - npm
```
Comment (alokedesai) [inline specs/ssh/generate_new_ssh_key_for_github.yaml:5]: same here about `github` versus `npm`

**Review by alokedesai — APPROVED** [review]
LGTM! Thank you for contributing!

**Comment by WebRuin** [comment]
Would you please add `hacktoberfest-approved'

**Comment by alokedesai** [comment]
@WebRuin Happy to add that label, but I believe we also need to opt-in the repo for hacktoberfest as well. Please correct me  if I'm wrong

File: specs/ssh/copy_ssh_key_for_github.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (alokedesai) [reply specs/ssh/copy_ssh_key_for_github.yaml]: nit: I don't think this is done yet?

**Comment by alokedesai** [comment]
Hey @WebRuin! Sorry for the late followup. Would you mind addressing any of the in-flight comments, would love to get all of these amazing workflows merged 🚢 Thanks again!

**Pull request merged**